src/restruct_poc/types.hpp

```diff
@@ -0,0 +1,41 @@
+#pragma once
+
+#include <Kokkos_Core.hpp>
+
+namespace oturb {
+
+static constexpr std::size_t LieAlgebraComponents = 6;
+static constexpr std::size_t LieGroupComponents = 7;
+static constexpr std::size_t VectorComponents = 3;
+
+static constexpr double Tolerance = 1.e-16;
+
+using LieAlgebraFieldView = Kokkos::View<double* [LieAlgebraComponents]>;
+using LieGroupFieldView = Kokkos::View<double* [LieGroupComponents]>;
+using VectorFieldView = Kokkos::View<double* [VectorComponents]>;
+
+using View1D = Kokkos::View<double*>;
+using View1D_Vector = Kokkos::View<double[VectorComponents]>;
+using View1D_LieAlgebra = Kokkos::View<double[LieAlgebraComponents]>;
+using View1D_LieGroup = Kokkos::View<double[LieGroupComponents]>;
```

Comment:
If you're not using these types anywhere in the code, let's just go ahead and remove them.